### ISSUE TITLE
default MAPQ-threshold

### ISSUE THREAD

**Issue by slbai01** [haowenz/chromap#140]
command help info: `--MAPQ-threshold INT      Min MAPQ in range [0, 60] for mappings to be output [30]`

In fact, when executing subcommand `hic`, the default parameter is `1`

**Comment by mourisl** [comment]
Yes, hic alignment is quite different, e.g. no mate information is utilized, so the quality score is generally lower. This is part of the "--preset hic". We will change the documentation later.
